homepage/models.py

```diff
@@ -109,11 +110,39 @@ def does_course_have_prerequisites(course):
 
 
 class AppUser(models.Model):
+    # This field is the built-in django user model
+    # We currently use the following attributes from the django user model: username, password, email, is_active
     user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
 
     def __str__(self):
         return self.user.username
 
+    def toggle_user_activation(self):
+        self.user.is_active = not(self.user.is_active)
+
+    @staticmethod
+    # Creates a new appUser (without superuser permmisions) and saves it in the DB
+    # The is_active attribute of AppUser.user is automaticly set to 'True' when creating a new AppUser
+    def create_appUser(username, email, password):
+        appUser = AppUser()
+        user = User.objects.create_user(username, email, password)
+        appUser.user = user
+
+        appUser.save()
+        return appUser
+
+    @staticmethod
+    def get_all_appUsers():

```

Comment:
python convention is:
https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables
```suggestion
    def get_all_app_users():
```

Reply:
changed